manifests/README.md

```diff
@@ -0,0 +1,14 @@
+A number of manifests which will be deployed during installation.
+
+Assumption is that the list of sorted manifests is applicable using:
+
+```bash
+for MANIFEST in $(ls -1 | sort -h);
```

Comment:
nit: I am being told that ls already sorts by alphabetical order so there is probably no need for explicit sort

Reply:
Well  - My main goal is to spell out what assumptions I made in the naming of these manifests.

But thanks.